net.sonobus.SonoBus.yaml

```diff
@@ -0,0 +1,40 @@
+app-id: net.sonobus.SonoBus
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '20.08'
+command: sonobus
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+# Jack
+  - --filesystem=xdg-run/pipewire-0
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+cleanup:
+  - /lib/libjack*
+  - /lib/jack
```

Comment:
If you update shared-modules to the latest version, you can drop this cleanup. It's a recent change (more recent than when I said it should be here)